server/src/js/components/map/outlook.js

```diff
@@ -230,22 +235,27 @@ function OutlookMap (mapId, options) {
 
   // Create MapContainer
   const container = new MapContainer(mapId, containerOptions)
+  const keyElement = container.keyElement
   const map = container.map
 
   //
   // Private methods
   //
 
-  // Outlook set day function
-  const setDay = (day) => {
-    // Set feature visibility
+  // Set feature visiblity
+  const setFeatureVisibility = () => {
     areasOfConcern.getSource().forEachFeature((feature) => {
-      const isVisible = parseInt(feature.get('day')) === parseInt(day)
+      const riskLevel = parseInt(feature.get('risk-level'))
```

Comment:
The parseInt method calls need to have the radix set to 10 as per answers here https://stackoverflow.com/questions/6611824/why-do-we-need-to-use-radix-parameter-when-calling-parseint this is to ensure that decimal numeration is used and parseInt doesn't make an incorrect assumption about the string being parsed

Reply:
radix number added to parseInt function